.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+*Short paragraph summarizing the changes*
+
+---
+
+Thank you for submitting a PR to xDSL!
+
+Make sure to follow the steps in the checklist below *before* you submit this PR!
+
+## Checklist before you submit this PR:
+
+ - [ ] If your PR is not ready for review, open it as a draft instead.
+ - [ ] Make sure your PR title follows the [guidelines](https://github.com/xdslproject/xdsl/wiki/Tags-for-commit-messages-and-PR-titles)
+ - [ ] Add some tags to categorize the PR
+ - [ ] Request review from people you *know* have to sign off on these changes, you can leave it blank for now if you are not sure.
```

Comment:
Can new users request reviewers? I guess that they don't need to and we'll do for them in that case. In any way, a new users wouldn't know who is responsible for what.

Reply:
Note: Pull request authors can't request reviews unless they are either a repository owner or collaborator with write access to the repository.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review